ERCs/eip-5732/contracts/CommitToMintImpl.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+// Author: Zainan Victor Zhou <zzn-ercref@zzn.im>
+// Visit our open source repo: http://zzn.li/ercref
+
+pragma solidity ^0.8.9;
+
+import "hardhat/console.sol";
```

Comment:
nit: shouldn't merge in `hardhat/console` import or usage, just for local testing

Reply:
Good catch. should remove